dockstore-webservice/src/main/java/io/dockstore/webservice/resources/DockerRepoResource.java

```diff
@@ -16,25 +16,6 @@
 
 package io.dockstore.webservice.resources;
 
-import java.util.ArrayList;
```

Comment:
It's hard to tell, but it looks like all the changes in this file are whitespace changes. If so, just revert this to make the pull request more understandable.